js/adslot.jsx

```diff
@@ -1,75 +1,80 @@
-
 import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
You don't need to do this; the change is already backward compatible. You do not access React.PropTypes at all so it is moot.